extensions/amazon-sam/deployment/pom.xml

```diff
@@ -20,23 +21,21 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-server-common-deployment</artifactId>
+            <artifactId>quarkus-vertx-web-deployment</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-amazon-lambda-resteasy</artifactId>
+            <artifactId>quarkus-resteasy-jsonb-deployment</artifactId>
```

Comment:
This project should not depend on Resteasy.  it should be an additional dependency that is added to the application's pom.xml file after the HTTP api is chose (quarkus-resteasy, quarkus-vertx-web, or quarkus-undertow)